app/models/game.rb

```diff
@@ -27,7 +29,14 @@ class Game < ActiveRecord::Base
   validates :playing_time, presence: true
   validates :complexity, presence: true
 
+
   def complexity_name
     COMPLEXITIES[complexity]
   end
+
+  def update_average(review=nil)
```

Comment:
Unused method argument - `review`. If it's necessary, use `_` or `_review` as an argument name to indicate that it won't be used. You can also write as `update_average(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.